specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2022-10-01/examples/ChildAvailabilityStatuses_List.json

```diff
@@ -0,0 +1,149 @@
+{
+  "parameters": {
+    "resourceUri": "resourceUri",
+    "api-version": "2022-10-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/227b734f-e14f-4de6-b7fc-3190c21e69f6/resourceGroups/JUHACKETRHCTEST/providers/Microsoft.Compute/virtualMachineScaleSets/rhctest/virtualMachines/4/providers/Microsoft.ResourceHealth/availabilityStatuses/current",
+            "name": "current",
+            "type": "Microsoft.ResourceHealth/AvailabilityStatuses",
+            "location": "southcentralus",
+            "properties": {
+              "availabilityState": "Available",
+              "summary": "There aren't any known Azure platform problems affecting this virtual machine",
+              "detailedStatus": "",
+              "reasonType": "",
```

Comment:
Curious question. How accurate is this. Would it actually be returned as empty, or would it just be omitted from the response?

Reply:
surprisingly yes